### DISCUSSION TITLE
如何确认我访问的某个网址是否走代理

### DISCUSSION THREAD

**Discussion by iyzh635428** [v2fly/v2ray-core#738] [Q&A] [answered]
由于v2ray不支持icmp协议，导致很多相关的测试工具没法使用，我现在有一个基本的测试需求，打开翻墙工具，我访问淘宝或某个任意网站的时候，想知道走的是代理还是直连，以确定我的v2ray配置是否生效，可至今没有什么好的方法，有大神可以指教一下吗？
有些网站倒是可以，比如百度，我只需搜索“ip”就可以知道我的访问ip是国内还是翻墙服务器的，一下子就可以知道我走的是代理还是直连。但其它网站就暂时没办法。

**Comment by Loyalsoldier** [comment, marked as answer]
`loglevel` 设置为 `debug`，查看 log 文件

**Reply by iyzh635428** [reply]
> `loglevel` 设置为 `debug`，查看 log 文件

感谢，我今天已经自己找出你所说的这种方法了，本想删除这贴子，结果找不到删除按钮。

**Reply by SyberRabbit** [reply]
感谢，解决了

**Reply by SyberRabbit** [reply]
@iyzh635428 你删了贴，可就帮不到别人了，比如我，不过谢谢没删贴，

**Reply by zyf0330** [reply]
v2ray 程序是否可以提供一个命令直接用来测试某个 url，毕竟修改 log 还是比较麻烦的

**Comment by Marco-YAO** [comment]
大佬，我也有这个疑问，包括app端的，能指点迷津吗？你们讲的我都不懂^_^
